crates/language-tests/src/tests/cmp.rs

```diff
@@ -189,8 +189,39 @@ impl RoughlyEq for Uuid {
 	}
 }
 
+/// Tolerance for floating point comparisons.
+const EPSILON: f64 = 1e-15;
+/// Tolerance for decimal comparisons.
+const EPSILON_DECIMAL: Decimal = dec!(1e-15);
```

Comment:
I kind of tried to make this test suite compile as fast as possible, trying to avoid as many procedural macros as possible. Is there a way we can have this value without resorting to a proc-macro?